small fixes to getting started docs

davekaj opened the pull request:
Small doc fixes that I noticed when at eth SF. 
- typos
- adding store.remove
- moved graph deploy AFTER createdb, because it should be like that

Jannis commented on docs/getting-started.md:
```diff
@@ -254,23 +259,25 @@ To get an IPFS daemon working locally, do the following:
+
+Deploying the subgraph means that the Subgraph is ingesting all the events from the blockchain, transforming the data with mappings, and storing it in the Graph Node. Note that running a subgraph can safely be stopped and started, and it will pick up where it has left off. 
+
+Now you can run `yarn deploy --watch` in the subgraph directory (make sure to pass the right host and port to 'graph deploy' via the '--ipfs' argument). `yarn deploy` should have been added to `package.json` in section 3.3 when you went through the [Graph CLI documentation](https://github.com/graphprotocol/graph-cli). This will compile the mappings, and deploy the mappings, schema and the subgrap ...
```
'graph deploy' -> backticks

Jannis commented on docs/getting-started.md:
```diff
@@ -304,8 +311,8 @@ Once you have the local node fully synced, you can run the following command:
   --postgres-url postgresql://<USERNAME><:PASSWORD>@localhost:5432/<POSTGRES_DB_NAME> \
-  --ethereum-rpc <ETHEREUM_NETWORK_NAME>:127.0.0.1:8545`
-  --ipfs 127.0.0.1:5001 \
+  --ethereum-rpc <ETHEREUM_NETWORK_NAME>:127.0.0.1:8545
```
This line needs a `\` at the end.

nenadjaja replied:
There should be a backward slash at the end of this line:
--ethereum-rpc <ETHEREUM_NETWORK_NAME>:127.0.0.1:8545 \

davekaj commented:
wait do not merge, i also need to add `yarn deploy` to after starting the graph node. otherwise you get the `HTTP error deploying the subgraph: ECONNREFUSED` error

nenadjaja commented on docs/getting-started.md:
```diff
@@ -254,23 +259,25 @@ To get an IPFS daemon working locally, do the following:
+
+Deploying the subgraph means that the Subgraph is ingesting all the events from the blockchain, transforming the data with mappings, and storing it in the Graph Node. Note that running a subgraph can safely be stopped and started, and it will pick up where it has left off. 
+
+Now you can run `yarn deploy --watch` in the subgraph directory (make sure to pass the right host and port to `graph deploy` via the '--ipfs' argument). `yarn deploy` should have been added to `package.json` in section 3.3 when you went through the [Graph CLI documentation](https://github.com/graphprotocol/graph-cli). This will compile the mappings, and deploy the mappings, schema and the subgrap ...
```
> in section 3.3 when you went through the [Graph CLI documentation](https://github.com/graphprotocol/graph-cli).

in section 3.3. This can be found in [Graph CLI documentation](https://github.com/graphprotocol/graph-cli).

nenadjaja commented:
I have 1 more small comment, otherwise it's good to go

davekaj commented:
I addressed nenas last comment, and then I moved deploy to step 4.4, for the reason i stated in the last comment. if that change looks good this can be merged

i also added in some line breaks throughout the doc so that it didn't have 400 character lines.

davekaj commented:
addressed the errors in the titles having too many ###
